math/mathmore/test/testGSLRootFinder.cxx

```diff
@@ -153,7 +153,7 @@ int testGSLRootFinder() {
 
 
 
-  // the following two examples won't work when interpreted CINT
+  // the following two examples won't work when interpreted Cling
```

Comment:
This is unlikely to be true for Cling.  i.e. I bet the example works fine in Cling ...